api/task_manager.cc

```diff
@@ -35,6 +35,9 @@ struct full_task_status {
     std::string module;
     tasks::task_id parent_id;
     tasks::is_abortable abortable;
+    const tasks::task_manager::foreign_task_vector& child_tasks;
+
+    full_task_status(const tasks::task_manager::foreign_task_vector& child_tasks_) : child_tasks(child_tasks_) {}
```

Comment:
I think it'd be better to copy the task ids here into a vector in full_task_status